boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts

```diff
@@ -76,6 +76,20 @@ arduino_serial: &uart2 {};
 
 &i2c4 {
 	status = "ok";
+
+	fxos8700@1f {
```

Comment:
@galak, how do you feel about adding this node if the sensor isn't populated on the board by default? Is there a dts convention for unpopulated board components? I already started down this path with the sensor interrupt gpios in commit ad52bce1d027a23392ab36d50bbfea24d0cfb2c0, but not sure how far we want to go with it. I'm a bit concerned we could confuse people that try to use the sensor and don't realize it's not actually there.

Reply:
I have to admit it confused me until @MaureenHelm told me to check if the sensor was populated on my board. Do we know if this concerns a limited set of boards? The fact sheet clearly list the FXOS8700CQ sensor, however it's not populated on the photos of the boards.

Reply:
> Do we know if this concerns a limited set of boards?

I'm looking into it. The A1 schematic implies that the sensor should be populated, whereas the A3 schematic marks it DNP. I have an A2 board. My rt1060 and rt1064 boards also don't have the sensor populated. It looks like DNP might be the theme for the sensor on newer rt boards, possibly for cost reasons.

Reply:
Thanks for the search. If newer rt boards do not have it, I agree that we should not provide the DTS entry for it. I have therefore pushed a new version of the PR dropping that commit.